.github/workflows/nx.template.yaml

```diff
@@ -57,7 +57,13 @@ jobs:
         run: bun i --lockfile package-lock.json
 
       - name: Evaluate affected projects
-        id: configure-nx
+        id: all-affected
```

Comment:
I would suggest using a different ID name, as this kinda gives the feeling that `all` plugins are included under this step @abrl91

Reply:
maybe just `calculate-affected`